### DISCUSSION TITLE
Context is not exported/importable

### DISCUSSION THREAD

**Discussion by oveddan** [wevm/wagmi#97] [General]
this library is awesome!

I have a case where I need to be able to grab the wagmi context and pass it down manually, because I'm using a third party lib that hijacks the context.  In particular, I want to use wagmi hooks in a 3d scene rendered by react three fiber.
A limitation of react three fiber is that you need to re-pass the context down to children components of the 3d scene.
See:
https://docs.pmnd.rs/react-three-fiber/advanced/gotchas#consuming-context-from-a-foreign-provider

So ideally I can do something like:

```jsx
import { useContextBridge } from "@react-three/drei"
import { Context as WagmiContext } from 'wagmi'

const SceneContainer = () => {
  const ContextBridge = useContextBridge(WagmiContext)
    return (
      <Canvas>
        <ContextBridge>
          <Scene />
        </ContextBridge>
      </Canvas>
    )
}
```

However `Context` is not currently exported so its not possible to do this.  

I can try doing:
`import { Context as WagmiContext } from "wagmi/dist/declarations/src/context";`
However I get an error: "Module not found: Package path ./dist/declarations/src/context is not exported from package"

Can this Context please be exported from the lib?

Thank you!

**Comment by tmm** [comment]
Done!
